File: core/src/com/symbol/game/ecs/component/KnockbackComponent.java
package com.symbol.game.ecs.component;

import com.badlogic.ashley.core.Component;
import com.badlogic.gdx.utils.Pool;

public class KnockbackComponent implements Component, Pool.Poolable {

    public boolean knockingBack = false;

    @Override
    public void reset() {
        knockingBack = false;
    }

}

File: core/src/com/symbol/game/ecs/component/KnockbackComponent.kt
package com.symbol.game.ecs.component

import com.badlogic.ashley.core.Component
import com.badlogic.gdx.utils.Pool

class KnockbackComponent : Component, Pool.Poolable {

    var knockingBack = false
    var timer = 0f

    override fun reset() {
        knockingBack = false
        timer = 0f
    }

}